_commands/basics/alias.md

```diff
@@ -0,0 +1,86 @@
+---
+---
+
+alias
+-------
+
+`alias` is a command to create shortcut to commands and applications in the terminal.
+
+~~~bash
+$ alias l="ls"
+$ l
+Documents  Downloads  Music     Public     Videos
+Desktop    Pictures   Templates
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Usage
+Things to note: when using alias, make sure that there is no space around the equal sign.
+~~~bash
+$ alias l = "ls"
+alias l='ls -CF'
+bash: alias: =: not found
+~~~
```

Comment:
Okay. I added all the newlines. I was looking at the markdown file through github and didn't see any formatting issues so I thought the formatting had no problem.